### ISSUE TITLE
facility to ignore specific examples

### ISSUE THREAD

**Issue by vgheo** [concordion/concordion#258]
As previously noted, concordion can ignore a full fixture by adding the junit `@Ignore` annotation to it.

In practice, it is often useful to ignore specific examples from one fixture.

Please add a way to ignore specific examples from one fixture.

This could be achieved by introducing of a new concordion-specific annotation for fixture classes like 
`@IgnoreExamples({"ex1", "ex2" })`

**Comment by nigelcharman** [comment]
There's an enhancement for this that works with Spring in #267, and we will be working to make sure this works with JUnit 5 before releasing it. If you would like to try the current implementation, please use [Jitpack](https://jitpack.io/) to add to your dependencies.

**Comment by nigelcharman** [comment]
Hi, the new implementation status of `Ignored` can be applied to specific examples. Sorry, I omitted to note that in the release notes. I've added it now :)

Individual examples can be ignored by marking up the example in the specification with a status of `Ignored`. For example, using Markdown format:

`### [Header text](- "example name c:status=Ignored")`

or in HTML:

`<div c:example="my-example" c:status="Ignored">`
